kolena/_api/v2/dataset.py

```diff
@@ -33,6 +34,7 @@ class RegisterRequest:
     name: str
     id_fields: List[str]
     uuid: str
+    sources: Optional[List[Dict[str, str]]]
```

Comment:
Should this have a `None` default value?

Reply:
i think we don't need the default value at here.
this is the caller side definition, and we already pass the `DEFAULT_SOURCES` as default value in the proper functions.
we have the default value on the server side for backward compatibility